### PR TITLE
Fix LAO launch bug with scala server

### LINKED ISSUE #963: [BUG] LAO launching not working on fe2 and be2 together
### Description (Actual behavior)
The java app cannot launch an LAO with the scala backend.

### How to reproduce
1. Initialize the wallet
2. Click on 'Launch' on the navbar
3. Type any name
4. Click on `Launch` button
5. See error as user is brought to the settings page instead of the LAO one.

### Version & Environment
This bug was reproduced on:
- Commit: [2a0317a](https://github.com/dedis/popstellar/commit/2a0317ae0bb284e6ab9a11446597833c342cde41)

##### Front-ends:
- [ ] Fe1-Web (please include browser's names & version)
- [x] Fe2-Android (please specify if phone or emulation, and Android version)
- [ ] Not applicable

##### Back-ends:
- [ ] Be1-Go
- [x] Be2-Scala
- [ ] Not applicable

##### Environment (as applicable):
 - Device: [Samsung S8]
 - OS: [Android 9]

### Workaround
There is no workaround.

### Impact
The app is unusable for the an organizer who wants to connect to a scala server.

### Possible root cause
The be2 seems to be fine as a catchup is received by fe2. It seems the fe2 has an error because it waits for some consensus communications from the backend.

### PR DESCRIPTION
**Pull request by Warhedgehog**
This fixes #963

### CODE REVIEW

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java [hunk 1]
```diff
@@ -109,7 +109,8 @@ public class LaoDetailViewModel extends AndroidViewModel
       new MutableLiveData<>();
   private final MutableLiveData<SingleEvent<Boolean>> mOpenNewRollCallEvent =
       new MutableLiveData<>();
-  private final MutableLiveData<SingleEvent<HomeViewModel.HomeViewAction>> mOpenRollCallEvent = new MutableLiveData<>();
+  private final MutableLiveData<SingleEvent<HomeViewModel.HomeViewAction>> mOpenRollCallEvent =
```
Comment (pierluca) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java:112]: In general, try to split in separate commits reformatting without code changes from actual code changes 😉 it makes it easier to review

**Review by Tesa27 — APPROVED** [review]

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java (hunk 1 above)
Comment (Warhedgehog) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailViewModel.java:112]: I'll try to be more careful. It was not on purpose, it's (a misuse of) auto-formatting

**Review by GabrielFleischer — COMMENTED** [review]
LGTM, great catch.

I think there are some other missuse of the `subscribe()` function without error handling in the code. It might be a good idea to check them out 😉

**Review by GabrielFleischer — APPROVED** [review]
Now with the approval.

**Pull request merged**